<?xml version='1.0' encoding='utf-8'?>
<log><entry><timestamp>2025-09-24T09:55:20.543315</timestamp><command>python</command><args>emulator.py --vfs ./my_vfs --log ./log.xml --script ./start.sh</args><error>Ошибка: Команда 'python' не найдена.</error></entry><entry><timestamp>2025-09-24T09:56:20.741303</timestamp><command>python</command><args>emulator.py</args><error>Ошибка: Команда 'python' не найдена.</error></entry><entry><timestamp>2025-09-24T09:56:51.117047</timestamp><command>python3</command><args>emulator.py</args><error>Ошибка: Команда 'python3' не найдена.</error></entry><entry><timestamp>2025-09-24T09:56:54.604436</timestamp><command>py</command><args>emulator.py</args><error>Ошибка: Команда 'py' не найдена.</error></entry></log>